images/miq-app/docker-assets/appliance-initialize.sh

```diff
@@ -24,13 +24,13 @@ restore_pv_data
 cd ${APP_ROOT}
 bin/rake evm:deployment_status
 
+# Generate httpd certificate
+/usr/bin/generate_miq_server_cert.sh
```

Comment:
Shouldn't this go away entirely if we are doing it in the apache pod? Or do we still need it here for some reason?